<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleBar</class>
 <widget class="QWidget" name="TitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>31</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>31</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>31</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">#iconButton, #minimizeButton, #maximizeButton, #closeButton, #homePushButton, #userPushButton,#helpPushButton {
  border-style: none;
  background-color: transparent;
}

#minimizeButton:hover, #maximizeButton:hover, #homePushButton:hover, #userPushButton:hover,#helpPushButton:hover{
  background-color: #80c7c7c7;
}

#minimizeButton:pressed, #maximizeButton:pressed, #homePushButton:pressed, #userPushButton:pressed,#helpPushButton:pressed {
  background-color: #80808080;
}

#closeButton:hover {
  background-color: #e81123;
}

#closeButton:pressed {
  background-color: #8c0a15;
}

#TitleBar
{
	background-color: #1A1A1A;
	border: none;
}

#titleLabel
{
	color: #9bcf50
}


</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="minimizeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Minimize</string>
     </property>
     <property name="icon">
      <iconset resource="FgHelpWindow.qrc">
       <normaloff>:/FgHelpWindow/icons/min.png</normaloff>:/FgHelpWindow/icons/min.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="maximizeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Maximize</string>
     </property>
     <property name="icon">
      <iconset resource="FgHelpWindow.qrc">
       <normaloff>:/FgHelpWindow/icons/max.png</normaloff>
       <normalon>:/FgHelpWindow/icons/restore.png</normalon>:/FgHelpWindow/icons/max.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset resource="FgHelpWindow.qrc">
       <normaloff>:/FgHelpWindow/icons/close.png</normaloff>:/FgHelpWindow/icons/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>46</width>
       <height>31</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FgHelpWindow.qrc"/>
 </resources>
 <connections/>
</ui>
